src/components/PopupWithImage.js

```diff
@@ -7,11 +7,17 @@ export default class PopupWithImage extends Popup {
     this._previewImageCaption = document.querySelector(".modal__preview-title");
   }
 
-  open({ name, link }) {
-    this._previewImageElement.alt = name;
-    this._previewImageCaption.textContent = name;
-    this._previewImageElement.src = link;
+  // open({ name, link }) {
+  //   this._previewImageElement.alt = name;
+  //   this._previewImageCaption.textContent = name;
+  //   this._previewImageElement.src = link;
+  //   super.open();
+  // }
+
+  open(cardData) {
+    this._previewImageElement.alt = "Image of ${cardData.name}";
```

Comment:
```js

"Image of ${cardData.name}";
```


There is a mistake: you need to correct it like that to insert the name of the image


```js
    this._previewImageElement.alt = `Image of ${cardData.name}`;


```

Reply:
I'm assuming I just change this to a single quote?

Reply:
> I'm assuming I just change this to a single quote?

no) you need to use special quotes called  **backtick (`) **




More info here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals